forms-flow-web/src/components/Dashboard/Dashboard.js

```diff
@@ -284,7 +284,7 @@ const Dashboard = React.memo(() => {
                           }
                           onChange={(e) => {
                             setShowClearButton(e.target.value);
-                            setSearchTextInput(e.target.value);
+                            setSearchTextInput(e.target.value.replace(/[^\w\s]/gi, ""));
```

Comment:
Check if the regex works in all the browsers. We had similar issue before

Reply:
replace() is supported by all modern web browsers, including Internet Explorer, Edge, Chrome, Firefox, and Safari.

Reply:
Check on ios device/ Safari . the regex pattern as such may have issue as well. The pattern format support may be different